config/file_config.go

```diff
@@ -276,6 +276,8 @@ type HoneycombLoggerConfig struct {
 
 type StdoutLoggerConfig struct {
 	Structured bool `yaml:"Structured" default:"false"`
+	SamplerEnabled	bool   `yaml:"SamplerEnabled" `
+	SamplerThroughput int    `yaml:"SamplerThroughput" default:"5"`
```

Comment:
This `SamplerThroughput` default value conflicts with the default set in `configMeta.yaml`.

Reply:
It also conflicts with the stated default value in the generated documentation.

Reply:
Good find! I honestly don't really understand the relationship here. I got the default from line 274, which is the config for `HoneycombLogger`. It looks like that one also has the same issue. 
@kentquirk , do you mind to explain a bit more on the relationship between each default values?

Reply:
Awesome! Then everything looks good to me.

Reply:
Please see #894 -- turns out there were a few of these.